Encryption.md

```diff
@@ -39,32 +39,32 @@ speed and access control granularity.
 2. Implement "client-side" encryption/decryption (storage client). The storage server 
 must not see plaintext data, metadata or encryption keys.
 3. Leverage authenticated encryption that allows clients to check integrity of the 
-retrieved data - making sure the file (or file parts) had not been replaced with a 
+retrieved data - making sure the file (or file parts) have not been replaced with a 
 wrong version, or tampered with otherwise.
 4. Support column access control - by enabling different encryption keys for different 
-columns, and for the footer.
+columns and for the footer.
 5. Allow for partial encryption - encrypt only column(s) with sensitive data.
 6. Work with all compression and encoding mechanisms supported in Parquet.
 7. Support multiple encryption algorithms, to account for different security and 
 performance requirements.
 8. Enable two modes for metadata protection:
    * full protection of file metadata
-   * partial protection of file metadata, that allows legacy readers to access unencrypted 
+   * partial protection of file metadata that allows legacy readers to access unencrypted 
  columns in an encrypted file.
-9. Miminize overhead of encryption: in terms of size of encrypted files, and throughput
+9. Minimize overhead of encryption: in terms of size of encrypted files, and throughput
 of write/read operations.
 
 
 ## Technical Approach
 
 Each Parquet module (footer, page headers, pages, column indexes, column metadata) is 
```

Comment:
The use of "module" is confusing later because it isn't used elsewhere it Parquet documentation. Please add a definition instead of using it and defining it at the same time.